### PR TITLE
sshKey addon can be managed in Admiral

### PR DESCRIPTION
**Pull request by deepikasl**
https://github.com/Shippable/admiral/issues/1157,
https://github.com/Shippable/admiral/issues/1158,
https://github.com/Shippable/admiral/issues/1159

Verified locally when `sshKeys` was already using `ssh-key`,
steps followed:
copied state.json to baseMigrations and ran:
```
$ sudo docker run -v /home/ubuntu/admiral:/home/shippable/admiral/ -v /etc/shippable:/etc/shippable drydock/microbase:master sh -c 'cd /home/shippable/admiral/baseMigrations && node --version && cd .. && npm install && cd - && npm install && node migrate.js'
```
In this case, it dint create a new shhKeys sysInt using `sshKey` int

screenshot from 2017-10-10 12 09 51

### CODE REVIEW

File: baseMigrations/migrate.js [hunk 1]
```diff
@@ -897,7 +897,7 @@ function _createSSHKeysSystemIntegration(bag, next) {
 
   var systemIntegration = {
     name: 'sshKeys',
-    masterName: 'ssh-key',
+    masterName: 'sshKey',
```
Comment (rageshkrishna) [inline baseMigrations/migrate.js]: How will this skip if the `sshKeys` system integration was already created using the `ssh-key` master integration?

File: baseMigrations/migrate.js [hunk 2]
```diff
@@ -905,6 +930,10 @@ function _createSSHKeysSystemIntegration(bag, next) {
         bag.stateJson.systemSettings.systemNodePrivateKey
     }
   };
+  // if sshKeys system integration is already using ssh-key masterInt
```
Comment (shrivara) [inline baseMigrations/migrate.js]: Why skip here? Can't we just skip the function entirely?

File: static/scripts/dashboard/dashboardCtrl.js [hunk 3]
```diff
@@ -2444,7 +2448,7 @@
 
       var bag = {
         name: 'sshKeys',
-        masterName: $scope.vm.installForm.sshKeys['ssh-key'].masterName,
+        masterName: $scope.vm.installForm.sshKeys['sshKey'].masterName,
```
Comment (shrivara) [inline static/scripts/dashboard/dashboardCtrl.js]: I'm not sure about how this works. We're not creating keys if `ssh-key` is already present. So, those keys will not show up anymore right?

File: baseMigrations/migrate.js (hunk 2 above)
Comment (shrivara) [reply baseMigrations/migrate.js]: Nevermind, I read this wrong.

**Comment by shrivara** [comment]
@deepikasl, could you please split this into multiple PRs for the different issues created? It will be easier to review if the PR is doing only one thing.

**Pull request closed without merging**